json-endpoints/class.wpcom-json-api-get-comments-tree-v1-1-endpoint.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+new WPCOM_JSON_API_Get_Comments_Tree_v1_1_Endpoint ( array(
+	'description' => 'Get a comments tree for site.',
+	'min_version' => '1.1',
+	'max_version' => '1.1',
+	'group'       => 'comments-tree',
+	'stat'        => 'comments-tree:1',
```

Comment:
Do we want to record a separate stat for new endpoint version?

Reply:
Hm, great question – other endpoints like `Jetpack_JSON_API_Plugins_Modify_v1_2_Endpoint` don't seem to: https://github.com/Automattic/jetpack/blame/b0cda18ec5710636fd03e7cc9f772d0430dfbdbc/json-endpoints/jetpack/class.jetpack-json-api-plugins-modify-v1-2-endpoint.php#L8

Is this intentional @enejb ?

Reply:
To be honest I don't quite know. I copied that from a different endpoint. 
cc @lezama Do we need to update it?

Reply:
what's the norm?

Reply:
> what's the norm?

Some endpoints in Dotcom record a separate stat for different versions, but I don't see a similar pattern in Jetpack code (the endpoints that do haven't been synced anyway). I think the question is more "do we need it"? 

Since we are going to provide upgrade nudges for Comments management in Calypso, I thought it might be useful to see if there are old Jetpack clients that are still using the old endpoint.

Reply:
The comments-tree v1 endpoint was never enabled on the client for Jetpack, since we were waiting for the v1.1 improvements.

Reply:
Oh right, in that case I guess we are fine with this in its present state.